components/RichText.tsx

```diff
@@ -21,19 +21,36 @@ export type RichTextProps = {
   wysiwyg: StoryblokRichtext;
   // "default" is for main content, e.g., Story body content
   type?: 'default' | 'card';
-  isLightText?: boolean;
+  textColor?: 'black' | 'white' | 'black-70';
+  bgColor?: 'black' | 'black-50' | 'black-60' | 'black-70' | 'white' | 'none';
   textAlign?: TextAlignType;
   className?: string;
 };
 
+const textClasses = {
+  black: 'text-gc-black',
+  white: 'text-white',
+  'black-70': 'text-black-70',
+};
```

Comment:
For the RichText component, instead of just light/dark text, added another black-70 option for rich text captions on the site